src/test/resources/resyncContext.xml

```diff
@@ -7,17 +7,18 @@
 
     <context:property-placeholder location="classpath:broker.properties"/>
 
-    <bean id="resyncService" class="org.esbtools.gateway.resync.ResyncService" >
-        <property name="resyncConfigurations">
-            <map>
-                <entry key="GitHub"><ref bean="gitHubSystemConfiguration"/></entry>
-            </map>
-        </property>
+    <bean id="gitHubResyncService" class="org.esbtools.gateway.resync.JmsResyncService">
+        <constructor-arg index="0" ref="gitHubSystemConfiguration" />
```

Comment:
You can omit index if you want; by default Spring will try to get the right args by type if there is no ambiguity